PrintTheReverseOfThreeDigitNumber.java

```diff
@@ -0,0 +1,23 @@
+import java.util.Scanner;
+public class PrintTheReverseOfThreeDigitNumber {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Input 3 digits number");
+        int ThreeDigitsNumber = sc.nextInt();
+        if (ThreeDigitsNumber > 999 || ThreeDigitsNumber < 100) {
+            System.exit(1);
+        } else {
```

Comment:
Related` if-else` combination is the same comment that I wrote above.

Reply:
will be fixed